spec.md

```diff
@@ -271,7 +271,7 @@ are built up from aggregating other types.
 ## Ground Types
 
 There are five ground types in FIRRTL: an unsigned integer type, a signed
```

Comment:
four ground types?

Reply:
Good catch. Fixed in the first `fixup!` commit.